kolibri/plugins/setup_wizard/assets/src/modules/pluginModule.js

```diff
@@ -6,7 +6,7 @@ import { FacilityImportResource } from '../api';
 
 const SetupStrings = createTranslator('SetupStrings', {
   personalFacilityName: {
-    message: 'Home Facility {name}',
+    message: 'Home Facility for {name}',
```

Comment:
This should be:

```
Home Facility for '{name}'
```

From the [design system](https://design-system.learningequality.org/writing/#notes):

> Always wrap user-generated text in 'single quotes'. Do not try to add formatting such as bold or italics